src/browser/api/window.js

```diff
@@ -763,6 +763,14 @@ Window.create = function(id, opts) {
         _window: browserWindow
     };
 
+    // Set initial preload state if window has preload script
+    if (_options.preload) {
+        winObj.preloadState = {
+            src: _options.preload,
```

Comment:
Why not `url` instead of `src`? I realize `src` is used in the DOM, but for consistency with _our_ usage elsewhere in core when we have URL string.